### ISSUE TITLE
MAS 1.3.5 - Identify Input Purpose

### ISSUE THREAD

**Issue by scottbommarito** [NuGet/NuGetGallery#7524]
We should verify our site uses the autocomplete attribute wherever it makes sense.

List of possible autocomplete attribute values: https://www.w3.org/TR/html52/sec-forms.html#autofill

For example, forms that accept email addresses should be configured to automatically fill in the user's email address.

**Comment by scottbommarito** [comment]
Our site doesn't appear to use `autocomplete` fields anywhere. We will need to add these fields where necessary.

Non-exhaustive list of places to add these attributes to:
- login form
- register form
- forgot password form
- forgot AAD form
- account settings email address change form (both user and org)
- add member to org
- add user ownership
